lib/vsc/install/shared_setup.py

```diff
@@ -1533,6 +1534,7 @@ def action_target(package, *args, **kwargs):
         'author': [sdw, ag, jt],
         'maintainer': [sdw, ag, jt],
         'install_requires': install_requires,
+        'data_files': [('etc', glob.glob('lib/etc/*'))],
```

Comment:
where does this place the files from an rpm? in `/etc`? if so, rename the file so this is clear this is the vsc-install prospector profile (no need to mention pylint or py3), ideally add it in a subdir.
but first try to see if you can avoid it